### PR TITLE
Added escape hatch feature

### PR DESCRIPTION
**Pull request by Dev-Mikko**
Added all the elements needed for the escape hatch feature on edumeet-client

### CODE REVIEW

File: src/Peer.ts [hunk 1]
```diff
@@ -267,6 +280,8 @@ export class Peer extends EventEmitter {
 			picture: this.picture,
 			raisedHand: this.raisedHand,
 			raisedHandTimestamp: this.raisedHandTimestamp,
+			escapeMeeting: this.escapeMeeting,
```
Comment (havfo) [inline src/Peer.ts]: This should not be part of the peerInfo. This gets propagated down to all other clients.

Comment (Dev-Mikko) [reply src/Peer.ts]: OK, got it. It will be fixed ASAP

File: src/Room.ts [hunk 2]
```diff
@@ -264,6 +267,26 @@ export default class Room extends EventEmitter {
 		return this.peers.items.filter((p) => p !== excludePeer);
 	}
 
+	@skipIfClosed
+	public escapeMeeting() {
```
Comment (havfo) [inline src/Room.ts]: This should be simplified down to:
```
if (!this.peers.items.some((peer) => !peer.escapeMeeting))
    this.close();
```

Comment (havfo) [reply src/Room.ts]: We may want to send the `escapeMeeting` notification to show some notification on the client, and then forcibly close the connection.

Comment (Dev-Mikko) [reply src/Room.ts]: I think it is good now

**Review by havfo — APPROVED** [review]
LGTM

**Pull request merged**